### DISCUSSION TITLE
GitHub Links for other git providers

### DISCUSSION THREAD

**Discussion by sceptri** [quarto-dev/quarto-cli#11400] [Feature Requests]
### Description

Right now, one can use the Website Navigation feature `repo-actions` to add GitHub links to the page sidebar. Although one can specify the repo URL, the links always follow the GitHub naming scheme, e.g. for "View source" we have:
- `https://github.com/<user>/<repo>/blob/<branch>/<filename>`

A similar feature would be handy for e.g. GitLab, Forgejo (Codeberg), which would look like this:
- `https://gitlab.com/<user>/<repo>/-/blob/<branch>/<filename>`
- `https://codeberg.org/<user>/<repo>/src/branch/<branch>/<filename>`

This could surely be done with some string formatting (or maybe in some cleaner way)

**Comment by mcanouil** [comment]
Thanks for the interest, but this is a duplicate.
Please search a bit before opening feature requests or bug reports.

- https://github.com/quarto-dev/quarto-cli/issues/5301

**Reply by sceptri** [reply]
Sorry I did try searching, but "repo-actions" seems to have been a really bad keyword. Will be better next time!
